src/types.ts

```diff
@@ -34,6 +34,15 @@ export interface UtxoInterface {
   txid: string;
   vout: number;
   asset?: string;
+  assetcommitment?: string;
```

Comment:
I'm not sure this type is the same type you are thinking of. AFAIK this is only used internally, it should not be used to  Esplora JSON response

Reply:
nvm, this is actually ok.